cqf-fhir-cr/src/test/java/org/opencds/cqf/fhir/cr/measure/r4/CareGaps.java

```diff
@@ -363,6 +332,31 @@ public SelectedMeasureReport measureReport() {
                     .get()
                     .getResource()));
         }
+
+        public SelectedBundle measureReportEvaluatedResourcesFound() {
+            // get resource References from the Patient Bundle
+            List<String> bundleResourceReferences = bundleReport().getEntry().stream()
+                    .map(g -> g.getResource()
+                            .getResourceType()
+                            .toString()
+                            .concat("/" + g.getResource().getIdPart()))
+                    .collect(Collectors.toList());
+            // get resource References from evaluatedResources on Measure Report
+            List<MeasureReport> measureReports = bundleReport().getEntry().stream()
+                    .filter(x -> x.getResource().getResourceType().toString().equals("MeasureReport"))
```

Comment:
Just curious why you did it this way.  Is this code meant to be called with multiple FHIR versions?